sdk/formrecognizer/ai-form-recognizer/review/ai-form-recognizer.api.md

```diff
@@ -551,12 +681,23 @@ export interface ListModelsOptions extends OperationOptions {
 export interface ListOperationsOptions extends OperationOptions {
 }
 
+// @public
+export interface ModelAdministrationOperationStateCommon {
+    apiVersion?: string;
+    createdOn: Date;
+    lastUpdatedOn: Date;
```

Comment:
In TA, we use `modifiedOn`, is that name applicable here by any chance?

Reply:
We've already GA'd this property name, unfortunately, so there's no room to align them.